library/Zend/Code/Reflection/ClassReflection.php

```diff
@@ -177,12 +177,17 @@ public function getMethods($filter = -1)
         return $methods;
     }
 
+    /**
+     * Returns an array of reflection classes of traits used by this class.
+     *
+     * @return array|null
+     */
     public function getTraits()
     {
         $vals = array();
         $traits = parent::getTraits();
-        if (! $traits) {
-            return;
+        if ($traits === null) {
```

Comment:
`$traits === null` is not possible here, because `ReflectionClass:getTraits()` returns `array` or `null` in case of an error (see [manual](http://php.net/manual/en/reflectionclass.gettraits.php)). `if (! $traits) {` is correct here.